src/net.h

```diff
@@ -436,19 +439,16 @@ struct CombinerAll
     }
 };
 
-// Signals for message handling
-struct CNodeSignals
+// Interface for message handling
```

Comment:
Ideally we'd use a doxygen-compliant comment here, so that it shows the class description
```
/**
 * Interface for message handling
 */
```

Reply:
I had to rebase anyway, so I went ahead and fixed these two nits as well.